src/resources/js/v2/tickets-commerce.js

```diff
@@ -1,3 +1,4 @@
+/* global tribe, tecTicketsCommerceData */
```

Comment:
these should be defined in the `.eslintrc` file

Reply:
Should we also move the localized vars to `.eslintrc` as well? i.e: `tecTicketsCommerceData` is the localized variable that only gets loaded with this script.

Reply:
If it was defined in another JS file but used here, then yes move it to `.eslintrc` so the linter knows to not flag that global variable.

Reply:
Nope, `tecTicketsCommerceData` is only used in this script.